src/main/java/org/sunbird/cassandra/utils/CassandraOperationImpl.java

```
@@ -194,7 +194,7 @@ public Long getRecordCount(String keyspace, String table) {
             Row row = connectionManager.getSession(keyspace).execute(selectQuery).one();
             return row.getLong(0);
         } catch (Exception e) {
-            throw e;
+        	return 0L;
```

Need to throw the exception - so that caller knows the issues...